### PR TITLE
SOC-1451 | Displaying deleted posts

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/SOC-1451

### CODE REVIEW

File: front/styles/main/component/_share-feature.scss [hunk 1]
```diff
@@ -2,6 +2,10 @@
 	display: block;
 	text-align: right;
 
+	.is-deleted & {
```
Comment (bkoval) [inline front/styles/main/component/_share-feature.scss]: ? :)

**Comment by bkoval** [comment]
LGTM :japanese_castle:

**Comment by rafalkalinski** [comment]
nice one :)
LGTM

**Comment by kvas-damian** [comment]
Will open new PR to dev

**Pull request closed without merging**